internal/fs/fs.go

```diff
@@ -1619,26 +1618,19 @@ func (fs *fileSystem) createLocalFile(
 		}
 	}()
 
-	fs.mu.Lock()
-
 	fullName := inode.NewFileName(parent.Name(), name)
 	child, ok := fs.localFileInodes[fullName]
```

Comment:
This requires fs lock. Ref: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1735

Seems we don't have concurrent test for this scenario. This would have caught with concurrent create and delete files test.